src/spec-node/devContainersSpecCLI.ts

```diff
@@ -610,7 +626,11 @@ async function doBuild({
 		await ensureNoDisallowedFeatures(buildParams, config, additionalFeatures, undefined);
 
 		// Support multiple use of `--image-name`
-		const imageNames = (argImageName && (Array.isArray(argImageName) ? argImageName : [argImageName]) as string[]) || undefined;
+		const duplicateImageNames = (argImageName && (Array.isArray(argImageName) ? argImageName : [argImageName]) as string[]) || undefined;
+		let imageNames = undefined;
```

Comment:
Hi @samruddhikhandale,
I have used this approach to check the list which we are passing does not have same image with differet tags

Reply:
@prathameshzarkar9 Looks like we have test failures post your latest commit, can you help fix those?